Avalonia.Templates.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <PackageType>Template</PackageType>
-    <PackageVersion>0.10.10</PackageVersion>
+    <PackageVersion>0.10.11-rc1</PackageVersion>
```

Comment:
i would just call it 0.10.11 because usually, we map packageversion and Avalonia version we use in templates,but for 0.10.11-rc1 I wouldnt update everything because its preview

Reply:
done